lib/modules/hosts/pixiv.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    moduleID: 'pixiv',
+    name: 'pixiv',
+    logo: '//www.pixiv.net/favicon.ico',
+    domains: ['pixiv.net'],
+    detect: ({ href }) => (/^https?:\/\/(?:www\.)?pixiv\.net\/member_illust\.php(?:[?&](?:mode=medium|illust_id=(\d+))){2,}).exec(href),
```

Comment:
`/^https?:\/\/(?:www\.)?pixiv\.net\/member_illust\.php.*[?&]illust_id=(\d+)/i` should work.

We can ignore the `mode=medium` since the id is all that really matters.